src/command.c

```diff
@@ -453,6 +456,60 @@ int cmd_proxy(struct command *cmd, struct redis_data *data)
 
     if (strcasecmp(type, "INFO") == 0) {
         return cmd_proxy_info(cmd);
+    } else if (strcasecmp(type, "SETREMOTES") == 0) {
```

Comment:
Yes, because later we may let more config options changable in run time. I think we should keep all this to only one `CONFIG SET` command.